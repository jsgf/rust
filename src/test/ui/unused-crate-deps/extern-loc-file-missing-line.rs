// --extern-location file: with no line number

// aux-crate:bar=bar.rs
// compile-flags:--extern-location bar=file:file.txt

#![warn(unused_crate_dependencies)]

fn main() {}
