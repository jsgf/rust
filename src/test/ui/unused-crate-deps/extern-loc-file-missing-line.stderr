error: `--extern-location`: missing line number on `file`

